package.json

```
@@ -13,7 +13,7 @@
     "Safari >= 12"
   ],
   "scripts": {
-    "test": "jest",
+    "test": "vitest",
```

I think this needs to be:
```suggestion
    "test": "vitest run",
```
since [Vitest runs in watch mode by default](https://vitest.dev/guide/cli.html#vitest) (so this task will never finish in the CI pipeline as it's currently written).

---

good spot but it does exit in CI pipeline https://github.com/Laings-Online-Limited/forms-admin/actions/runs/9366514608/job/25784128335?pr=1#step:11:1 

As it js testing and lining etc are run now together as part of vitest there's no real need to have this step in CI https://github.com/alphagov/forms-admin/blob/main/.github/workflows/test.yml#L69-L70

---

https://vitest.dev/guide/cli.html#vitest

>vitest
>Start Vitest in the current directory. Will enter the watch mode in development environment and **run mode in CI automatically.**

---

Ooh, smart!